sales: updated get paid to validate doc for RST adjustments (#4461)
odoo/documentation: content/applications/sales/sales/send_quotations/get_paid_to_validate.rst

Enable online payment
=====================

Go to :menuselection:`Sales app --> Configuration --> Settings`, scroll to the :guilabel:`Quotations
& Orders` heading, and activate the :guilabel:`Online Payment` feature by checking the box next to
it, and clicking :guilabel:`Save`.

.. image:: get_paid_to_validate/online-payment-setting.png
   :align: center
   :alt: How to enable online payment on Odoo Sales.

After checking the box beside the :guilabel:`Online Payment` feature, a link to the
:guilabel:`Payment Acquirers` appears beneath it. Click that link to reveal the entire
:guilabel:`Payment Acquirers` page.

.. image:: get_paid_to_validate/payment-acquirers.png
   :align: center
   :alt: Payment acquirers page in the Odoo Sales application.

On the :guilabel:`Payment Acquirers` page, payment acquirers can be selected and configured. Before
creating (or modifying) a payment acquirer, be sure to review the documentation related to the ways
payment acquirers can be used with Odoo, such as:

- :doc:`/applications/finance/payment_acquirers/paypal`
- :doc:`/applications/finance/payment_acquirers/authorize`

> - :doc:`/applications/finance/payment_acquirers`

Reviewer: These links should be updated, the finance docs have been updated to "payment_providers"

Author: Actually, in v14, this folder is referred to as "payment_acquirers" -- but I totally know that it has since changed. However, since this is for v14, I think it'll be best to leave it as-is, so there won't be any broken links

=================================
Online payment order confirmation
=================================

With Odoo *Sales*, online payments can be used to get automatic confirmation on orders.

Enable online payment
=====================

> Go to :menuselection:`Sales app --> Configuration --> Settings`, scroll to the
> :guilabel:`Quotations & Orders` heading, and activate the :guilabel:`Online Payment` feature by
> checking the box next to it, and clicking :guilabel:`Save`.

Reviewer 2: Early line break
```suggestion
Go to :menuselection:`Sales app --> Configuration --> Settings`, scroll to the :guilabel:`Quotations
& Orders` heading, and activate the :guilabel:`Online Payment` feature by checking the box next to
it, and clicking :guilabel:`Save`.
```

> :guilabel:`Quotations & Orders` heading, and activate the :guilabel:`Online Payment` feature, by
> checking the box next to it, and clicking :guilabel:`Save`.

Reviewer 3: Remove commas
```suggestion
:guilabel:`Quotations & Orders` heading, and activate the :guilabel:`Online Payment` feature by
checking the box next to it and clicking :guilabel:`Save`.
```

.. image:: get_paid_to_validate/online-payment-setting.png
   :align: center
   :alt: How to enable online payment on Odoo Sales.

> After checking the box beside the :guilabel:`Online Payment` feature, a link to the
> :guilabel:`Payment Acquirers` appears beneath it. Clicking that link reveals the entire
> :guilabel:`Payment Acquirers` page.

Reviewer 3: ```suggestion
After checking the box beside the :guilabel:`Online Payment` feature, a link to the :guilabel:`Payment Acquirers` page appears beneath it.
```

Author: I understand cutting that last portion to be more succinct, but since it is necessary to proceed with the flow, I think I'm going to keep it in, just to avoid any confusion. But, again, I totally get where you're coming from with this suggestion.

.. image:: get_paid_to_validate/payment-acquirers.png
   :align: center
   :alt: Payment acquirers page in the Odoo Sales application.

On the :guilabel:`Payment Acquirers` page, payment acquirers can be selected and configured. Before
creating (or modifying) a payment acquirer, be sure to review the documentation related to the ways
payment acquirers can be used with Odoo, such as:

- :doc:`/applications/finance/payment_acquirers/paypal`
- :doc:`/applications/finance/payment_acquirers/authorize`
- :doc:`/applications/finance/payment_acquirers`

.. note::

>    On quotation templates, the :guilabel:`Online Payment` feature is located under the
>    :guilabel:`Confirmation` tab.
> 
>    .. image:: get_paid_to_validate/confirmation-tab.png
>       :align: center
>       :alt: Confirmation tab with the online payment feature.
> 
>    On standard quotations, the :guilabel:`Online Payment` feature is located under the
>    :guilabel:`Other Info` tab.
> 
>    .. image:: get_paid_to_validate/other-info.png
>       :align: center
>       :alt: Other Info tab with the online payment feature.

Reviewer 2: These two images are not entirely necessary, I think your text is sufficient enough. Can remove these two if you agree :) (don't forget to `git rm` the media files)
```suggestion
   On quotation templates, the :guilabel:`Online Payment` feature is located under the
   :guilabel:`Confirmation` tab.

   On standard quotations, the :guilabel:`Online Payment` feature is located under the
   :guilabel:`Other Info` tab.
```

Register a payment
==================

> After opening quotations from their received email, customers are presented with different options

Reviewer 3: ```suggestion
After opening quotations from the email they receive, customers are presented with different options
```

to make online payments, in the :guilabel:`Pay with` section.

.. image:: get_paid_to_validate/validate-order.png
   :align: center
   :alt: How to register a payment on Odoo Sales.

.. tip::
   Clicking the :guilabel:`Customer Preview` smart button on the quotation form provides a preview

>    of the quotation that the customer sees, along with the various online payment options they can

Reviewer 3: ```suggestion
   of the quotation as the customer sees it, along with the various online payment options they can
```

   choose from.

.. seealso::
   - :doc:`/applications/sales/sales/send_quotations/quote_template`

>    - :doc:`/applications/finance/payment_acquirers`
>    - :doc:`/applications/finance/payment_acquirers/paypal`
>    - :doc:`/applications/finance/payment_acquirers/authorize`

Reviewer 2: I feel we can remove these since they were stated previously, however, it doesnt hurt to leave them– up to you :)
```suggestion
```
